rlImGui/rlImGui.cs

```diff
@@ -27,14 +28,36 @@ public static class rlImGui
 
         private static ImGuiMouseCursor CurrentMouseCursor = ImGuiMouseCursor.COUNT;
         private static Dictionary<ImGuiMouseCursor, MouseCursor> MouseCursorMap = new Dictionary<ImGuiMouseCursor, MouseCursor>();
-        private static KeyboardKey[]? KeyEnumMap;
-
         private static Texture2D FontTexture;
 
+        private static IntPtr IconFontRanges = IntPtr.Zero;
```

Comment:
I suggest moving IconFontRanges and IconFontData to its own file like you have for IconsFontAwesome6.cs. Makes it easier for people to choose if they want to include/swap out the IconFont.

Reply:
Moved it to the icons file